wasm-rpc/src/bincode.rs

```diff
@@ -4,6 +4,26 @@ use bincode::enc::Encoder;
 use bincode::error::{AllowedEnumVariants, DecodeError, EncodeError};
 use bincode::*;
 
+impl Encode for Uri {
+    fn encode<E: Encoder>(&self, encoder: &mut E) -> Result<(), EncodeError> {
+        self.value.encode(encoder)
+    }
+}
+
+impl Decode for Uri {
+    fn decode<D: Decoder>(decoder: &mut D) -> Result<Self, DecodeError> {
+        let value = String::decode(decoder)?;
+        Ok(Uri { value })
+    }
+}
+
+impl<'de> BorrowDecode<'de> for Uri {
+    fn borrow_decode<D: BorrowDecoder<'de>>(decoder: &mut D) -> Result<Self, DecodeError> {
+        let value = String::borrow_decode(decoder)?;
+        Ok(Uri { value })
+    }
+}
```

Comment:
So only codec for Uri was missing for deriving bincode:* for `Value`?

Reply:
Yes, it does not have any other custom types.